certified-connectors/HTML To PDF By Pascalcase/apiDefinition.swagger.json

```diff
@@ -0,0 +1,90 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "HTML to PDF by Pascalcase",
+    "description": "Converts HTML files to PDF. No more 2mb limitation.",
+    "version": "1.0",
+    "contact": {
+      "name": "pascalcase",
+      "url": "https://pascalcase.com/Home/Contact",
+      "email": "support@pascalcase.com"
+    }
+  },
+  "host": "pascalcaseapi.azurewebsites.net",
+  "basePath": "/api/htmltopdf",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [],
+  "paths": {
+    "/": {
+      "post": {
+        "responses": {
+          "default": {

```

Comment:
@VikrantUpadhyay1 It is not allowed to have a default response only. Please define a proper response. Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses).